src/main/groovy/org/exoplatform/platform/am/settings/PlatformSettings.groovy

```diff
@@ -134,46 +134,27 @@ class PlatformSettings {
     }
     this.librariesDirectory.eachFile(findFilenameClosure)
     if (fileFound == null) {
-      throw new ErroneousSetupException(
-          "Erroneous setup, Unable to find edition jar in ${librariesDirectory}")
+      throw new ErroneousSetupException("Erroneous setup, Unable to find edition jar in ${librariesDirectory}")
     } else {
-      if (fileFound.contains("plf-enterprise-edition-service")){ // eXo EE
-        this.distributionType = DistributionType.ENTERPRISE
-      } else if (fileFound.contains("plf-community-edition-service")) { // Meeds
-        this.distributionType = DistributionType.COMMUNITY
-      } else if (fileFound.contains("platform-community-edition-service")) { // eXo CE
-        this.distributionType = DistributionType.EXO_COMMUNITY
-      } else {
-        this.distributionType = DistributionType.UNKNOWN
-      }
-      if (DistributionType.UNKNOWN.equals(this.distributionType)) {
-        throw new ErroneousSetupException("Erroneous setup, cannot compute the distribution type.")
-      }
       JarFile jarFile = new JarFile(fileFound)
       JarEntry jarEntry = jarFile.getJarEntry("conf/platform.properties")
       InputStream inputStream = jarFile.getInputStream(jarEntry)
       Properties platformProperties = new Properties()
       platformProperties.load(inputStream)
       this.version = platformProperties.getProperty("product.version")
+      try {
+        String distributionTypeFromProps = platformProperties.getProperty("product.distributionType")
+        if(!distributionTypeFromProps) {
+          this.distributionType = DistributionType.UNKNOWN
```

Comment:
we may need to delete the usage of enum for distribution type to avoid adding exo distributions in Meeds-io codebase.
IMO, we should make it as a simple string.

Reply:
@boubaker I will consider this to improve it in a later PR as I need to merge and test this PR

Reply:
Well I will continue working to implement your feedback, I hope this will be the last one for this PR :)